src/main/java/racingcar/entity/RacingCars.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.entity;
+
+import racingcar.strategy.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+    private final List<RacingCar> racingCars;
+
+    public RacingCars(List<Name> carNames) {
+        this.racingCars = new ArrayList<>();
+        initialize(carNames);
```

Comment:
```suggestion
        this.racingCars = initialize(carNames);
```
요런식으로 바로 선언해주면 더 명확할 것 같아요ㅎㅎ